index.html

```diff
@@ -142,19 +143,20 @@ <h2 class="card-title">Facebook 360</h2>
           <!--end of work nº 3-->
                 <!--work nº 4-->
                 <div class="card">
-                  <img class="card-picture" src="/images/picture4.PNG" alt="work-example"/>
+                  <img class="card-picture" src="/images/picture3.PNG" alt="work-example"/>
```

Comment:
The image for this card doesn't match the one from the template design. Kindly fix this.